config/manifests-rhoam/bases/managed-api-service.clusterserviceversion.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
```

Comment:
do we need the skip range here ?

Reply:
not really, that's a base and is fine as it is

Reply:
@MStokluska